docs/src/paradox/02-knora-ontologies/knora-base.md

```diff
@@ -286,6 +286,14 @@ Knora values are versioned. Existing values are not modified. Instead, a
 new version of an existing value is created. The new version is linked
 to the old version via the `previousValue` property.
 
+Since each value version has a different IRI, there is no IRI that can
+be used to cite the value, such that it will always refer to the latest
+version of the value. Therefore, the latest version of each value has
+a separate UUID, as the object of the property `valueHasUUID`. When
+a new version of the value is created, this UUID is moved to the new
+version. This makes it possible to cite the latest version of a value
+by searching for the UUID.
+
```

Comment:
There's no special logic for this. It gets the previous version of the value the same way that it would get it if you requested a resource with a timestamp without specifying a value UUID.

Specifying a UUID just adds a FILTER in the SPARQL query:

https://github.com/dhlab-basel/Knora/blob/2da6419cc62bf4e6aa92bf352c308732b3549565/webapi/src/main/twirl/queries/sparql/v2/getResourcePropertiesAndValuesGraphDB.scala.txt#L168

I haven't documented how that query gets previous versions of values using timestamps, would you like me to open a separate issue for that?

Reply:
> I haven't documented how that query gets previous versions of values using timestamps, would you like me to open a separate issue for that?

Maybe it could be useful in the design docs, what do you think?

Reply:
#1342

Reply:
thanks